config.toml

```
@@ -76,18 +76,15 @@ fullversion = "v1.18.0"
 version = "v1.18"
 githubbranch = "master"
 docsbranch = "master"
-deprecated = false
+deprecated = true
```

/hold

```suggestion
deprecated = false
```

commit before merging